.github/workflows/tests.yml

```diff
@@ -18,10 +19,21 @@ jobs:
             repository: opensafely/gh-testing-research
             path: gh-testing-research
       - name: Run action
+        # It will fail on sending data to honeycomb because we're providing a 
+        # dummy api key, but the extracted_stats file should be created if it completed 
+        # the rest of the steps
         uses: ./
+        continue-on-error: true
```

Comment:
I'm a bit nervous about this line because it changes the behaviour of the whole "Run action" test. Is it possible to instead make the "push to honeycomb" command conditional on `$HONEYCOMB_API_KEY != 'dummy-key'`?

I guess the ideal thing would be to handle this in the docker image itself so the test would still exercise the process of launching the docker container, it's just that it wouldn't actually try to push any data if it was using a dummy API key.

I'm thinking of something like:
```
CMD [ "/bin/sh", "-c", "[ $HONEYCOMB_WRITE_KEY = dummy-key ] || ./honeytail \
            --parser json \
            --writekey $HONEYCOMB_WRITE_KEY \
            --file $LOG_FILENAME \
            --dataset $DATASET \
            --backfill" ]
```
But there might be neater ways of doing it.

Reply:
:+1: I had tried to find a way to make it skip the honeycomb bit but I hadn't thought of putting it in the docker image